student.rb

```diff
@@ -2,7 +2,7 @@
 require './classroom'
 
 class Student < Person
-  attr_reader :classroom
+  attr_accessor :classroom
```

Comment:
- [ ] (OPTIONAL) Classroom should remain an `attr_reader` since we have a method dedicated to adding new classrooms 👍